style.css

```diff
@@ -42,4 +42,54 @@ h1 {
 h2{
     line-height: 0;
     font-size: 1.3em;
-}
\ No newline at end of file
+}
+
+table {
+  width: 100%;
+  table-layout: fixed;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+th, td {
+  width: 33.33%; 
+  text-align: left;
+  position: relative;
+  padding: 10px; 
+}
+
+th {
+  border-bottom: 3px solid black; 
+  font-weight: bold; 
+  background-color: #d3d3d3;
+}
+
+td {
+  border-bottom: 1px solid #ccc;
+  padding-right: 30px;
+}
+
```

Comment:
```suggestion

tr {
//padding here
}

```

Can you add more padding to the bottom and top so that the text is more centered in the box?

<img width="626" alt="Screenshot 2025-02-20 at 6 58 58 AM" src="https://github.com/user-attachments/assets/e130f143-01e3-40ca-8156-137ca9f73afa" />